packages/magento-cart-checkout/queries/BillingPage.graphql

```diff
@@ -7,5 +7,7 @@ query BillingPage($cartId: String!) {
     ...ShippingAddress
     ...Coupon
     ...PaymentMethodContext
+    #Retrieve cart items here so we can use the data on the succes page. This will put the cart items in the cache
+    ...CartItems
```

Comment:
Remove comment please/

Might it be an idea to create a OrderSuccessPage fragment to communicate the intent?

Reply:
We use the CartItemSummary fragment to retrieve the CartItems. We just need to add it to the billingPage fragment so it gets put into the cache. So that the CartItemSummary fragment can retrieve it from the cache

Reply:
Correction: we are using CartItemSummary **query** to retrieve the success page. I understand the caching situation, so that is great, therefor it might be a good idea to create a OrderSuccessPage fragment and use that in the CartItemSummary query and in this file

Reply:
Ahh okay, now I understand. I will create the OrderSuccesPageFragment